telegram/bot.py

```diff
@@ -2841,6 +2841,44 @@ def set_chat_permissions(self, chat_id, permissions, timeout=None, **kwargs):
 
         return result
 
+    @log
+    def set_chat_administrator_custom_title(self,
+                                            chat_id,
+                                            user_id,
+                                            custom_title,
+                                            timeout=None,
+                                            **kwargs):
+        """
+        Use this method to set custom title for owner or administrators. The bot must be an
```

Comment:
According to the official docs, bot's can only use this for admins promoted by the bot. Our docs should refelct this